conans/test/integration/toolchains/meson/test_test.py

```diff
@@ -0,0 +1,61 @@
+import os
+import pytest
+import textwrap
+
+from conans.test.assets.sources import gen_function_cpp
+
+
+from ._base import TestMesonBase
+
+
+@pytest.mark.tool_pkg_config
+class MesonTest(TestMesonBase):
+    _test_package_meson_build = textwrap.dedent("""
+    project('test_package', 'cpp')
+    hello = dependency('hello', version : '>=0.1')
+    test_package = executable('test_package', 'test_package.cpp', dependencies: hello)
+    test('test package', test_package)
+    """)
+
+    _test_package_conanfile_py = textwrap.dedent("""
+    import os
```

Comment:
indent more, the contents of the file cannot be aligned with the definition of the variable

Reply:
won't `textwrap` indent it properly?

Reply:
I indented one more level to right, still I don't know if it's relevant at all, since:
1. the textwrap should re-indent the code properly anyway
2. this is how pycharm indents by default
3. there are no errors using this indentation - e.g. code runs and compiles properly (usually python is very sensitive to indentation, but as soon as it's inside string constant I think it is safe)

Reply:
It was just style guide, I guess depending on the IDE configuration, it indent things differently. Up to my understanding, the left alignment should belong to statements. Everything that is not a statement should be indented. In this case, the statement that defines ``_test_package_conanfile_py = textwrap...`` continues in multiple lines. The continuation lines should be indented, to be able to distinguish them visually. Just my opinion, you can leave it as-is until we come up with a explicit style guide.